dev_ops/sql/CreateDatabase.sql

```diff
@@ -1 +1,9 @@
-CREATE DATABASE StreamerContractsDB;
\ No newline at end of file
+CREATE DATABASE StreamerContractsDB;
+
+DROP USER dbadmin;
+
+CREATE USER dbadmin WITH ENCRYPTED PASSWORD 'password';
```

Comment:
super-nit/optional: I think "application_user" is a little better of a name since we are enforcing restrictions on the user connecting from the application and not necessarily a dbadmin

Reply:
Sure I can change it. They will technically have admin privileges but application_user probably convey what it is better.